src/aap_eda/core/tasking/__init__.py

```diff
@@ -47,17 +49,35 @@
 ]
 
 
+def get_redis_prefix():
+    for _ in range(50):
+        try:
+            return f"EDA-{service_id()[0:8]}-rq"
+        except ProgrammingError:
+            # service_id() will fail at the first migration when
+            # dab_resource_registry_serviceid has not been created
+            if (
+                len(sys.argv) == 2
+                and sys.argv[0].endswith("aap-eda-manage")
+                and sys.argv[1] in ["migrate", "makeimgrations"]
```

Comment:
I think we need more commands to be handled, we don't want to wait for the service id and fail when running "showmigrations"